### DISCUSSION TITLE
Is vercel's official website open source?

### DISCUSSION THREAD

**Discussion by lianchi** [vercel/vercel#6080] [Help] [answered]
[vercel](https://vercel.com/) website is beautiful, is this site open source?

**Comment by markozxuu** [comment, marked as answer]
Hi @lianchi 👋 , unfortunately not, although Vercel shows the design guide of its components, the code of the site is private. It would be a good idea to link it to this discussion, in case in some future some components are OSS 😄 

https://github.com/vercel/vercel/discussions/4116
